research/berry/berrylib/fast_inla.py

```diff
@@ -321,7 +322,7 @@ def step(args):
         hess_inv = jax_faster_invert(diag, shift)
```

Comment:
Now that we're no longer using `hess_inv`, is there any room to get a little more performance by doing a hessian inverse-vector product instead of explicitly computing the hessian inverse. That is: can we combine 
```
hess_inv = jax_faster_invert(diag, shift)
step = -hess_inv.dot(grad)
```
into a single step that's faster? Looking at jax_faster_invert, I think the answer is yes! just inline `grad` to create a new `jax_faster_solve(diag, shift, grad)` function.

Reply:
OK but now we have a problem -- this is getting too fast 😃